sdks/python/apache_beam/pipeline.py

```diff
@@ -491,6 +492,11 @@ def apply(self, transform, pvalueish=None, label=None):
       finally:
         transform.label = old_label
 
+    # Attempts to alter the label of the transform to be applied only when it's
+    # a top-level transform.
```

Comment:
Please append: "This is so that the cell number will not be prepended to every child transform in a composite."

Reply:
Thanks, yes, adding it.